### PR TITLE
Documentation fixes for 2017 hackathon

### PR DESCRIPTION
**Pull request by bdwill**
### Proposed changes

Fix screenshots per issue #2126

Fix rendering issue with inline code snippet #1990

Fix for stack yaml reference (#1476)

Add shutdown procedures for #1017

### CODE REVIEW

**Comment by joaofnfernandes** [comment]
@bdwill looks like you have to rebase since everyone was rushing to fix the same issues.
Also, can you edit this PR's description, with just
```
Fix <issue-id>
```
That way, the [issue is automatically closed](https://help.github.com/articles/closing-issues-via-commit-messages/) once this PR is merged.

File: docker-cloud/apps/stack-yaml-reference.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 ---
-description: Stack YAML reference
-keywords: YAML, stack, reference
+description: Stack YAML reference for Docker Cloud
+keywords: YAML, stack, reference, docker cloud
 redirect_from:
 - /docker-cloud/feature-reference/stack-yaml-reference/
-title: Cloud stack file YAML reference
+title: Docker Cloud Stack File YAML Reference
```
Comment (mstanleyjones) [inline docker-cloud/apps/stack-yaml-reference.md]: Please fix the title to be sentence case here (Reference should be lowercase)

File: docker-cloud/apps/stack-yaml-reference.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 ---
 
-A stack is a collection of services that make up an application in a specific environment. Learn more about stacks [here](stacks.md). A **stack file** is a file in YAML format that defines one or more services, similar to a `docker-compose.yml` file but with a few extensions. The default name for this file is `docker-cloud.yml`.
+A stack is a collection of services that make up an application in a specific environment. Learn more about stacks for Docker Cloud [here](stacks.md). A **stack file** is a file in YAML format that defines one or more services, similar to a `docker-compose.yml` file for Docker Compose but with a few extensions. The default name for this file is `docker-cloud.yml`.
 
-## Stack file example
+*Looking for information on stack files for Swarm? Look* [here](/compose/compose-files/index.md).
```
Comment (mstanleyjones) [inline docker-cloud/apps/stack-yaml-reference.md]: Please no italics here. Maybe just make the link text `[Looking for information about stack files for swarms?]`

File: docker-for-mac/osxfs.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ you may experience exceptional, adequate, or poor performance with `osxfs`, the
 file system server in Docker for Mac. File system APIs are very wide (20-40
 message types) with many intricate semantics  involving on-disk state, in-memory
 cache state, and concurrent access by multiple  processes. Additionally, `osxfs`
-integrates   a mapping between macOS's FSEvents API and Linux's  inotify API
+integrates   a mapping between macOS's FSEvents API and Linux's `inotify` API
```
Comment (mstanleyjones) [inline docker-for-mac/osxfs.md]: nit: double space

**Comment by jmaitrehenry** [comment]
Fixes made on https://github.com/docker/docker.github.io/pull/2971

**Comment by mstanleyjones** [comment]
Merged via #2971

**Pull request closed without merging**